ui-tests/src/test/resources/features/connections/oauth.feature

```diff
@@ -145,8 +150,10 @@ Feature: Connections - OAuth
     And wait until integration "OAuth-Gcalendar-test" gets into "Running" state
 
     When sleep for "6000" ms
+#    Integration does not work: ENTESB-11282:
```

Comment:
seems like all the tests passed, is this remark really true?

Reply:
removed